web/main.py

```diff
@@ -16,3 +18,26 @@ async def root():
         keys[elem] = redis_database.get(elem)
 
     return {"message": "Hello World", "redis_database": keys}
+
+
+@app.post('/runners/refresh')
+async def refresh_data(request: Request):
+    try:
+        runners = github_manager.get_runners()
+        runner_m.update_all_runners(runners['runners'])
+    except Exception as e:
+        logger.info(e)
+    return Response(status_code=200)
+
+
+@app.post('/runners/reset')
+async def reset_reset_runners(request: Request):
+    runner_m.remove_all_runners()
+    g_runners = github_manager.get_runners()
+    runner_m.update_all_runners(g_runners['runners'])
+
+
+@app.post('/webhook')
+async def webhook_post(payload: str = Form(...), request: Request = None):
+    Webhook(payload=json.loads(payload), event=request.headers['X-Github-Event'])()
+    return Response(status_code=200)
```

Comment:
OPTIONAL: if you add something like this:
```suggestion
    return Response(status_code=200)


@app.delete('/runners/<runner name>')
async def delete_runner(request: Request):
    runner_m.delete_runner(runner.name)
    return Response(status_code=202)
```

I'll buy you a beer :beers: :smile: